RunCat365/Program.cs

```diff
@@ -158,6 +146,20 @@ public RunCat365ApplicationContext()
             current = 1;
         }
 
+        private ToolStripMenuItem ChargeToolStripMenu<T>(
```

Comment:
The name `ChargeToolStripMenu` is uncomfortable, so rename it to `CreateMenuFromEnum`.